packages/code-du-travail-frontend/src/search/SearchResults/Results.tsx

```diff
@@ -57,6 +57,7 @@ type ListLinkProps = {
   showTheme?: boolean;
   query?: string;
   centerTitle?: boolean;
+  noReport?: boolean;
```

Comment:
```suggestion
  disableAnalytics?: boolean;
```

Je trouve le `noReport` un peu trop générique.